html5ever/src/serialize/mod.rs

```diff
@@ -46,14 +46,14 @@ impl Default for SerializeOpts {
 }
 
 #[derive(Default)]
-struct ElemInfo {
+pub struct ElemInfo {
```

Comment:
Why does this need to be public?

Reply:
A good catch, it actually doesn't and I can't recall what made me turn it public. I restored the original visibility now.